components/builder-api/src/server/resources/origins.rs

```diff
@@ -1261,7 +1412,7 @@ fn list_origin_members(req: HttpRequest,
                        -> HttpResponse {
     let origin = path.into_inner();
 
-    if let Err(err) = authorize_session(&req, Some(&origin)) {
+    if let Err(err) = authorize_session(&req, Some(&origin), RoleAction::Read, None) {
```

Comment:
Should listing require membership?

Reply:
This does require membership. Authorize session still check's if you're a member as it did before. The new arguments are for verifying your level of access.